tests/Unit/VoyagerTest.php

```diff
@@ -64,5 +83,7 @@ public function testEachDimmerGroupHasAMaxAmountOfThreeDimmers()
         $dimmers = Voyager::dimmers();
 
         $this->assertEquals(2, count($dimmers));
+        $this->assertEquals(3, $dimmers[0]->count());
```

Comment:
This assertion proves what you were trying to prove in the previous test, so I think the other one can be removed.